guide/14-deep-learning/panoptic_segmentation_with_maxdeeplab.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>Here, <code>data</code> is the databunch prepared for training using the <code>prepare_data</code> method in the earlier steps. </p><p>For more information about the API, please see the <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#maxdeeplab" target="_blank">API reference</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>